etc/profile-a-l/cmake.profile

```diff
@@ -1,12 +1,15 @@
-# Firejail profile for cargo
-# Description: The Rust package manager
+# Firejail profile for cmake
+# Description: A cross-platform open-source make system
 # This file is overwritten after every install/update
 quiet
 # Persistent local customizations
-include cargo.local
+include cmake.local
```

Comment:
OT: It's always interesting how often you forget to change stuff if you do the same task multiple times.

Reply:
Very true :-) Not checking up on your work or anything. Firejailing the build systems had been on my to-do list for a while and I started to implement them one by one only recently. Thanks for the reviews, always a pleasure!